src/DefaultBuilder/samples/SampleApp/Program.cs

```diff
@@ -14,17 +15,14 @@ namespace SampleApp
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
-        }
+            await using var webApp = WebApplication.Create(args);
```

Comment:
Idk if this is too late, but why is this Create instead of CreateAsync?

Reply:
This doesn't do anything asynchronous.

Reply:
Why await then?

Reply:
Async disposable.

Reply:
man... I get it but I don't like it.

Reply:
It's not Create that is async. `WebApplication` is `IAsyncDisposable`.